backend/Dockerfile

```diff
@@ -51,7 +51,9 @@ RUN if [ "${ENABLE_PS1_SUPPORT}" = "YES" ] || [ "${ENABLE_GC_WII_SUPPORT}" = "YE
     dpkg --add-architecture i386 && apt-get update && \
     apt-get install -y -o APT::Immediate-Configure=false \
         wine; \
-    fi
+    fi && \
+    mkdir -p /wine && \
+    WINEPREFIX=/wine winecfg
```

Comment:
I think `mkdir -p /wine` should always be run, but `WINEPREFIX=/wine winecfg` should only be run in the body of the `if`? (Maybe move the `mkdir` before the `if`?)

Reply:
good point!